frontend/app/routes/protected/person-case/privacy-statement.tsx

```diff
@@ -0,0 +1,132 @@
+import { data, useFetcher } from 'react-router';
+import type { RouteHandle } from 'react-router';
+
+import { Trans, useTranslation } from 'react-i18next';
+import * as v from 'valibot';
+
+import type { Route, Info } from './+types/privacy-statement';
+
+import { requireAuth } from '~/.server/utils/auth-utils';
+import { i18nRedirect } from '~/.server/utils/route-utils';
+import { confirmPrivacyNoticeSchema } from '~/.server/validation/confirmPrivacyNoticeSchema';
+import { Button } from '~/components/button';
+import { ButtonLink } from '~/components/button-link';
+import { ErrorSummary } from '~/components/error-summary';
+import { InputCheckbox } from '~/components/input-checkbox';
+import { Progress } from '~/components/progress';
+import { getFixedT } from '~/i18n-config.server';
+import { handle as parentHandle } from '~/routes/protected/layout';
+import { getLanguage } from '~/utils/i18n-utils';
+
+export const handle = {
+  i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
+} as const satisfies RouteHandle;
+
+export async function loader({ context, request }: Route.LoaderArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const t = await getFixedT(request, handle.i18nNamespace);
+
+  return {
+    documentTitle: t('protected:privacy-statement.page-title'),
+    defaultValue: {
+      confirmPrivacyNotice: context.session.inPersonSINCase?.confirmPrivacyNotice,
+    },
+  };
+}
+
+export const meta: Route.MetaFunction = ({ data }) => {
+  return [{ title: data.documentTitle }];
+};
+
+export async function action({ context, request }: Route.ActionArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const lang = getLanguage(request);
+
+  const formData = await request.formData();
+  const input = { confirmPrivacyNotice: formData.get('confirmPrivacyNotice') as string };
+
+  // Validation schema
```

Comment:
This comment is not helpful -- I think `schema` is self-explanatory. But if you disagree, then just use `const validationSchema` instead of adding a comment.